crypto/hpke/hpke_util.c

```diff
@@ -397,6 +407,27 @@ EVP_KDF_CTX *ossl_kdf_ctx_create(const char *kdfname, const char *mdname,
     return kctx;
 }
 
+/*
+ * @brief look for a label into the synonym tables, and return its id
+ * @param st is the string value
+ * @param synp is the synonyms labels array
+ * @param outsize is the previous array size
```

Comment:
outsize is not a great name. 
is the number of elements in synp.

Reply:
sure s/outsize/arrsize/